include/xtensor-fftw/xtensor-fftw_config.hpp

```diff
@@ -9,8 +9,27 @@
 #ifndef XTENSOR_FFTW_CONFIG_HPP
 #define XTENSOR_FFTW_CONFIG_HPP
 
+#include <iostream>
```

Comment:
Maybe we should only include `<iostream>` if `XTENSOR_FFTW_DISABLE_EXCEPTIONS` is defined.

Reply:
I will be fine with that, this include will not be need else. But maybe it's overkill ? As you decide I will change

Reply:
I think that including all `iostream` just to get xtensor should not be required. I would prefer this to be in the `#if defined` section where you do the `std::cerr`.

Sorry for all the nitpicking!

Reply:
I do small cl for nitpicking :) I changed the code